internal/backends/python/python.go

```diff
@@ -248,6 +248,9 @@ func searchPypi(query string) []api.PkgInfo {
 		query = renamed[0]
 	}
 	results, err := SearchPypi(query)
+	if len(results) == 0 {
```

Comment:
shouldn't this be `if err != nil`

Reply:
unfortunately no, since SearchPypi returns an empty array here